docs/conf.py

```diff
@@ -150,7 +149,7 @@
 # If true, the index is split into individual pages for each letter.
 #html_split_index = False
 
-# If true, links to the reST sources are added to the pages.
+# If true, links to the rest sources are added to the pages.
```

Comment:
[reStructuredText](https://en.wikipedia.org/wiki/ReStructuredText) is sometimes abbreviated as RST, ReST or reST. "rest" is not one of the common abbreviations.